DependencyInjection/EzSystemsRecommendationExtension.php

```diff
@@ -8,13 +11,11 @@
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 use Symfony\Component\DependencyInjection\Loader;
 
-/**
- * This is the class that loads and manages your bundle configuration.
- *
- * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
- */
```

Comment:
Don't like this description, it's very generic and added by default by bundle generator.
I'll add it again.